modules/objectdb_profiler/editor/objectdb_profiler_panel.cpp

```diff
@@ -0,0 +1,471 @@
+	snapshot_list->connect("item_edited", callable_mp(this, &ObjectDBProfilerPanel::_edit_snapshot_name));
+	snapshot_list->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	snapshot_list->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	snapshot_list->set_anchors_preset(LayoutPreset::PRESET_FULL_RECT);
+
+	snapshot_list->set_allow_rmb_select(true);
+	snapshot_list->connect("item_mouse_selected", callable_mp(this, &ObjectDBProfilerPanel::_snapshot_rmb));
+
+	rmb_menu = memnew(PopupMenu);
+	add_child(rmb_menu);
+	rmb_menu->connect(SceneStringName(id_pressed), callable_mp(this, &ObjectDBProfilerPanel::_rmb_menu_pressed).bind(false));
+
+	HBoxContainer *diff_button_and_label = memnew(HBoxContainer);
+	diff_button_and_label->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	snapshot_column->add_child(diff_button_and_label);
+	Label *diff_against = memnew(Label(TTRC("Diff Against:")));
+	diff_button_and_label->add_child(diff_against);
+
+	diff_button = memnew(OptionButton);
+	diff_button->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	diff_button->connect(SceneStringName(item_selected), callable_mp(this, &ObjectDBProfilerPanel::_show_selected_snapshot).unbind(1));
+	diff_button_and_label->add_child(diff_button);
+
+	// Tabs of various views right for each snapshot.
+	view_tabs = memnew(TabContainer);
+	root_container->add_child(view_tabs);
+	view_tabs->set_custom_minimum_size(Size2(300 * EDSCALE, 0));
+	view_tabs->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	view_tabs->connect("tab_changed", callable_mp(this, &ObjectDBProfilerPanel::_view_tab_changed));
+
+	add_view(memnew(SnapshotSummaryView));
+	add_view(memnew(SnapshotClassView));
+	add_view(memnew(SnapshotObjectView));
+	add_view(memnew(SnapshotNodeView));
+	add_view(memnew(SnapshotRefCountedView));
+	add_view(memnew(SnapshotJsonView));
+
+	set_enabled(false);
+
+	// Load all the snapshot names from disk.
+	Ref<DirAccess> snapshot_dir = _get_and_create_snapshot_storage_dir();
+	if (snapshot_dir.is_valid()) {
+		for (const String &file_name : snapshot_dir->get_files()) {
+			Vector<String> name_parts = file_name.split(".");
+			ERR_CONTINUE_MSG(name_parts.size() != 2 || name_parts[1] != "odb_snapshot", "ObjectDB snapshot file did not have .odb_snapshot extension. Skipping: " + file_name);
+			_add_snapshot_button(name_parts[0], snapshot_dir->get_current_dir().path_join(file_name));
+		}
+	}
+}
+
+void ObjectDBProfilerPanel::add_view(SnapshotView *p_to_add) {
+	views.push_back(p_to_add);
+	view_tabs->add_child(p_to_add);
+	_update_view_tabs();
+}
+
+void ObjectDBProfilerPanel::_update_view_tabs() {
+	bool has_snapshot = current_snapshot.is_valid();
+	for (int i = 1; i < view_tabs->get_tab_count(); i++) {
+		view_tabs->set_tab_disabled(i, !has_snapshot);
+	}
+
+	if (!has_snapshot) {
+		view_tabs->set_current_tab(0);
+	}
+}
+
+void ObjectDBProfilerPanel::_update_diff_items() {
+	diff_button->clear();
+	diff_button->add_item("none", 0);
```

Comment:
```suggestion
	diff_button->add_item(TTRC("none"), 0);
```
Though not sure what is this list. If other items don't need to be translated, you could also add `diff_button->set_item_auto_translate_mode(AUTO_TRANSLATE_ALWAYS)` and disable auto-translation for `diff_button`.

Reply:
This is basically a list of filenames, that the user can rename, so this "none" option (now "None") is the only thing that should be translated, and the rest of the option labels should not be.

I set the `OptionButton` to be `AUTO_TRANSLATE_MODE_DISABLED` instead, and then used `TTR("None")` for that one option instead.

Let me know if there's some better way of doing it.